web/src/utils/factory-management/buildings.ts

```diff
@@ -45,18 +45,18 @@ export const calculateBuildingsAndPower = (factory: Factory) => {
     if (!factory.buildingRequirements[building.name]) {
       factory.buildingRequirements[building.name] = {
         name: building.name,
-        amount: '0',
+        amount: 0,
         powerPerBuilding: building.powerPerBuilding,
-        totalPower: '0',
+        totalPower: 0,
       }
     }
 
     const facBuilding = factory.buildingRequirements[building.name]
 
-    facBuilding.amount = String(parseFloat(facBuilding.amount) + parseFloat(building.amount))
-    facBuilding.totalPower = String(parseFloat(facBuilding.totalPower) + parseFloat(building.totalPower))
+    facBuilding.amount = facBuilding.amount + building.amount
+    facBuilding.totalPower = factory.totalPower + building.totalPower
```

Comment:
Looks like the behaviour changed here, so it used to use the facBuilding, now it's using the factory total?

Reply:
I didn't mean to change the behavior - this is a bug and we need to squash it. 🐞 

```suggestion
    facBuilding.totalPower = facBuilding.totalPower + building.totalPower
```

Reply:
Does that fix it @Maelstromeous?

Reply:
Yes - I didn't intend to create the bug, but it's still a bug, and regressions annoy me. :)